info-car-api/src/types/exam_schedule.rs

```diff
@@ -17,7 +17,7 @@ pub struct Exam {
     #[serde(deserialize_with = "deserialize_number_from_string")]
     /// Exam price
     pub amount: i32,
-    pub additional_info: String,
+    pub additional_info: Option<String>,
```

Comment:
Is this deserialised correctly when  `additional_info` is an empty string?

Reply:
Cannot confirm, because all my api responses contain nulls